src/resources/addys/claimables/query.ts

```diff
@@ -32,7 +32,7 @@ export type ClaimablesArgs = {
 // Query Key
 
 export const claimablesQueryKey = ({ address, currency }: ClaimablesArgs) =>
-  createQueryKey('claimables', { address, currency }, { persisterVersion: 3 });
+  createQueryKey('claimables', { address, currency }, { persisterVersion: 4 });
```

Comment:
any migration needed?

Reply:
i dont think so, this data is only stored in the react query cache which automatically gets wiped by this change